paddle/fluid/platform/mkldnn_utils.h

```diff
@@ -0,0 +1,72 @@
+/* Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#pragma once
+#include <mkldnn.h>
+#include <string>
+
+#ifdef PADDLE_WITH_MKLDNN
```

Comment:
Why `mkldnn_helper.h` and `mkldnn_reuse.h` do not need this?

Reply:
@tensor-tang Good catch. mkldnn_utils, should not have mentioned line either. This line was removed.